data/_ui/help.txt

```diff
@@ -91,7 +91,7 @@ help "map advanced shops"
 	`You can also press F to search the shipyard and outfitter maps for a specific ship or outfit.`
 
 help "map advanced stars"
-	`The starry map mode toggles the display of star types on the system map. This can also be triggered by pressing A while the map is open. The stars present in a system affect the amount of solar power and solar wind available for use by solar collectors and ramscoops respectively.`
+	`The starry map mode toggles the display of star types on the system map. This can also be toggled by pressing T. The stars present in a system affect the amount of solar power and solar wind available for use by solar collectors and ramscoops respectively.`
```

Comment:
You can't actually *toggle* the starry map mode by pressing 't', it's a one-way transition. This should be changed.